shell/components/nav/Header.vue

```diff
@@ -390,6 +387,31 @@ export default {
       }
 
       return null;
+    },
+
+    /**
+     * Store current path before redirect to logout page
+     */
+    storeCurrentPath() {
+      const isLogin = this.$route.name === 'auth-login' || this.$route.path === '/login'; // Cover dashboard and case of log out from ember;
+      const isLogout = this.$route.name === 'auth-logout';
+
+      // Prevent loop to login/logout pages
+      if (!isLogin && !isLogout) {
+        window.localStorage.setItem(BACK_TO, window.location.pathname);
+      }
+    },
```

Comment:
Is it sufficient to store the current path in the header when logout is clicked? Do we need to consider cases where the session expires via `auth-user-session-ttl-minutes`?

Reply:
That's a good point, but I do not know if it was implemented before.
I would rather handle that when we refactor the whole logic and take it out from the components in https://github.com/rancher/dashboard/issues/13939
This issue touched already too much unnecessary parts.

Reply:
Wait why do we have a test `auth-user-session-ttl-minutes` which is not failing?

Reply:
Also is it actually working on `master`? 🤔